app/assets/javascripts/modules/master_overlay.coffee

```diff
@@ -24,6 +25,7 @@ do (document, window, $=jQuery, config=@edsc.config, plugin=@edsc.util.plugin, p
       @contentHeightChanged()
 
     maximize: ->
+      @root.trigger('edsc.overlaychange')
```

Comment:
These work coincidentally. Also need to hide overlays whenever the level changes.